### DISCUSSION TITLE
V1.22 427 blt hs fw issue?

### DISCUSSION THREAD

**Discussion by okcub2010** [Jyers/Marlin#377] [Q&A] [answered]
I tried the 427 blt HS 3x3 and 5x5 fw tonight on a new 427 board in my older e3 pro. It flashed ok and after setting x and y probe offset and x offset I tried a test print. All went well except for both fw's the probe does a double probe and full retract for each probe. I thought the HS versions would keep the probe extended to speed up probing. 
I'll mention my BL touch is a older V3.0 Is possible that's the problem?
My e3 V2 422 board printer using Smith3D fast 3x3 blt fw does leave the probe extended for a double tap.
That blt in a V3.0.
I could try it on the 427 board if the jyers HS versions are ok?

Update Tried the newer v3.0 and it works ok. So it seems there was a blt fw change just before the v3.1 was released and I got lucky. so I'll be buying a new v 3.1 and saving the older 3.0 as a spare.

**Comment by Jyers** [comment, marked as answer]
Glad you found the issue!
